app/screens/NewScreen.tsx

```diff
@@ -0,0 +1,98 @@
+import { NativeStackNavigationProp } from "@react-navigation/native-stack";
+import { useNavigation } from "expo-router";
+import React from "react";
+import { StyleSheet, Text, TouchableOpacity, View } from "react-native";
+import { HomeStackParamList } from "../stacks/HomeStack";
+
+type HomeScreenNavigationProp = NativeStackNavigationProp<HomeStackParamList>;
+
+const NewScreen = () => {
+  const navigation: any = useNavigation<HomeScreenNavigationProp>();
```

Comment:
Import useNavigation from import { useNavigation } from '@react-navigation/native';

And do not ue expo-router if not needed, if you already have react-navigation